docs/features/while.md

```diff
@@ -0,0 +1,44 @@
+# While, break, and continue
+
+While loops are great for running a peice of code over and over again until a condition is met, here is one that waits for the user to type hello:
+```js
+while ( SystemIO.inp("Please type `hello`: ")! /= "hello" ) {}
+```
+
+The other main situations while loops find themselves in are times when a user wants to run a set of code indefinitely:
+```js
+while (true) {
+	print("hi")
+}
+```
+This ends up filling the screen with `hi`, but it can also be used for more productive measures as well.
```

Comment:
```suggestion
This ends up filling the screen with `hi`, but it can be used for more productive measures as well.
```

(redundancy with "as well")